crates/sui-core/src/epoch/reconfiguration.rs

```diff
@@ -26,7 +30,9 @@ pub trait Reconfigurable {
     fn recreate(channel: tonic::transport::Channel) -> Self;
 }
 
-const WAIT_BETWEEN_EPOCH_TX_QUERY_RETRY: Duration = Duration::from_millis(300);
+// TODO: Move these constants to a control config.
+const WAIT_BETWEEN_EPOCH_QUERY_RETRY: Duration = Duration::from_millis(300);
```

Comment:
I am keen to think of more "reactive" ways to initiate queries, by which authorities can register with others for events, and they get updates, instead of trying blindly on a timer. Not for this PR but lets think about how, and add a TODO here.